subsys/net/l2/ppp/ipcp.c

```diff
@@ -139,11 +139,7 @@ static int ipcp_config_info_req(struct ppp_fsm *fsm,
 	if (nack_idx > 0) {
 		struct net_buf *nack_buf;
 
-		if (count_rej > 0) {
-			code = PPP_CONFIGURE_REJ;
-		} else {
-			code = PPP_CONFIGURE_NACK;
-		}
+		code = PPP_CONFIGURE_REJ;
```

Comment:
It seems that the variable `count_rej` is now useless and can be removed. It is only incremented, but never read

Reply:
Good point, I will remove it